app/femr/ui/views/trauma/index.scala.html

```diff
@@ -138,7 +138,7 @@ <h4>Vitals</h4>
                                     <input type="number" step="any" min="0" class="fInput" id="@viewModel.getVitalNames.get(3).getName" name="@viewModel.getVitalNames.get(3).getName" placeholder="%"/>
                                 </div>
                                 <div class="vitalWrap">
-                                    <label for="heightFeet">Height</label>
+                                    <label for="height">Height</label>
```

Comment:
Why was this changed? Looks like `heightFeet` is used in many places in the code when I do `git grep heightFeet`.